dump25/sitelinks/tab.py

```
@@ -0,0 +1,129 @@
+def read_lines():
+    print("def read_lines():")
+    # ---
+    jsonfiles = list(parts_dir.glob("*.json"))
+    print(f"all json files: {len(jsonfiles)}")
+    # ---
+    cc = 0
+    # ---
+    for x in tqdm.tqdm(jsonfiles):
+        lines = get_lines(x)
+        # ---
+        for cc, line in enumerate(lines, start=cc):
```

Ruff (0.8.2)

Loop control variable `cc` overrides iterable it iterates

(B020)